content/communityExtensions/lottieTexture.md

```diff
@@ -0,0 +1,61 @@
+## Introduction
```

Comment:
Would be great to add some metadata so that the page will be indexed correctly. You can do that using a yaml object at the top of the page. Look at other .md files to get a few examples of how to do it. The important parts are the title, description and keywords.